.github/workflows/publish.yaml

```diff
@@ -44,8 +44,8 @@ jobs:
           # manylinux docker image, but I haven't figured out how to install CUDA on manylinux.
           os: [ubuntu-20.04]
           python-version: ['3.9', '3.10', '3.11', '3.12', '3.13']
-          torch-version: ['2.1.2', '2.2.2', '2.3.1', '2.4.0', '2.5.1', '2.6.0.dev20241001']
-          cuda-version: ['11.8.0', '12.3.2']
+          torch-version: ['2.1.2', '2.2.2', '2.3.1', '2.4.0', '2.5.1', '2.6.0', '2.7.0.dev20250130']
```

Comment:
pytorch 2.6.0 is not yet officially released

Reply:
pytorch 2.7.0.dev20250130 does not yet exist

Reply:
I prefert to keep this... why? Release Day (1/29/25)
pytorch 2.7 will have cuda 12.8:
https://github.com/pytorch/pytorch/issues/145570#issuecomment-2614110642
https://github.com/pytorch/pytorch/issues/145544#issuecomment-2613158998

Reply:
solved! Can you merge? pytorch 2.7.0 dev and 2.6.0 is out